src/styles/transitions.js

```diff
@@ -0,0 +1,10 @@
+const defaultTiming = '0.5s';
+const bgTransition = `background ${defaultTiming}`;
+const colorTransition = `color ${defaultTiming}`;
+const defaultTransition = `${bgTransition}, ${colorTransition}`;
+
+export default {
+    DEFAULT: defaultTransition,
+    COLOR: colorTransition,
+    BACKGROUND: bgTransition
+};
```

Comment:
Add a newline please =)
Isn't prettier working with you?

Reply:
Sure!
It works. Actually, I forgot to run the format 😅 

I'm also adding a .editorconfig file on this PR :)